v3/plugins/eclipse/che-theia/next/meta.yaml

```diff
@@ -21,6 +21,16 @@ spec:
         secure: true
         cookiesAuthEnabled: true
         discoverable: false
+   -  name: "webviews"
+      public: true
+      targetPort: 3100
+      attributes:
+        protocol: http
+        type: ide
```

Comment:
Type should be webview instead of ide

Reply:
See comment: https://github.com/eclipse/che/pull/15432#issuecomment-570613304

Reply:
Good catch. Thanks @pradeepmalji!
PR has been updated.